landingPage/assets/css/styles.css

```diff
@@ -0,0 +1,10 @@
+#user-posts {
+  display: inline-flex;
+  flex-wrap: nowrap;
+  margin-left: 10px;
+  margin-right: 10px;
```

Comment:
Same comment here, except use mr-2
For example <section id='user-posts' class='col-12 ml-2'>